src/klein/test/test_memory.py

```diff
@@ -38,9 +38,10 @@ def test_interfaceCompliance(self):
         store = MemorySessionStore()
         verifyObject(ISessionStore, store)  # type: ignore[misc]
         verifyObject(
-            ISession, self.successResultOf(  # type: ignore[misc]
+            ISession,  # type: ignore[misc]
+            self.successResultOf(
                 store.newSession(True, SessionMechanism.Header)
-            )
+            ),
```

Comment:
This is so that when black reformats it, we the `# type: ignore` comment for `ISession` stays with it.